css-positioning/absolute.css

```diff
@@ -0,0 +1,44 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+  position: relative;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  left: 240px;
+  top: 20px;
+
+  /* your code here */
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 165px;
+  left: 8px;
+  width: 300px;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
```

Comment:
please add position: absolute, as well as top, right, bottom, and left properties to make the gradient show up over the image